cypress-ete.sh

```diff
@@ -23,5 +23,5 @@ else
   yarn build
   yarn start &
   yarn wait-on:server
-  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/*.js"]}'
+  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/*.ts"]}'
```

Comment:
Mahesh is also changing in #1109, which version do we want to go in? https://github.com/guardian/gateway/pull/1109/files#diff-d2ddbe5aa010ad73e29564ba8183233bcec8b3b6982bdb6fff47c8a82f0c7143R26

Reply:
the version I use also matches subdirectories, which we want, could add it in here if this PR goes in first?

```suggestion
  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/**/*.ts"]}'
```

Reply:
will do 👌